### PR TITLE
GitHub action

### PR DESCRIPTION
**Pull request by wm-ismail-didouh**
# Goal of this PR :checkered_flag: 
Build source code and upload it to S3

# Proposed Changes :hammer_and_wrench:
* Create a workflow that does this for us on every release

### CODE REVIEW

File: .github/workflows/deploy-s3-bucket.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: cache-node-modules
+        uses: actions/cache@v1
+        id: cache-node-modules
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('yarn.lock') }}
```
Comment (wm-joan-mon) [inline .github/workflows/deploy-s3-bucket.yml]: this should be changed by package-lock.json

File: .github/workflows/deploy-s3-bucket.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        uses: jakejarvis/s3-sync-action@v0.5.1
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          AWS_S3_BUCKET: "wm-jitsi-meet"
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "eu-west-1"
+          SOURCE_DIR: "s3-package"
+          DEST_DIR: "${GITHUB_SHA}"
```
Comment (wm-joan-mon) [inline .github/workflows/deploy-s3-bucket.yml]: https://github.community/t/how-to-get-just-the-tag-name/16241
replace `"${GITHUB_SHA}"` with `${GITHUB_REF##*/}`

**Review by wm-joan-mon — APPROVED** [review]

File: .github/workflows/deploy-s3-bucket.yml (hunk 2 above)
Comment (wm-ismail-didouh) [reply .github/workflows/deploy-s3-bucket.yml]: Ouhhhh this is really cool

File: .github/workflows/deploy-s3-bucket.yml (hunk 1 above)
Comment (wm-ismail-didouh) [reply .github/workflows/deploy-s3-bucket.yml]: indeed, thanks !

**Pull request merged**